README.md

```diff
@@ -1,586 +1,15 @@
-# GEDI Subsetting
+# GEDI Subsetter
 
-- [Algorithm Outline](#algorithm-outline)
-- [Algorithm Inputs](#algorithm-inputs)
-  - [Specifying an AOI](#specifying-an-aoi)
-  - [Specifying a DOI](#specifying-a-doi)
-    - [L1B](#l1b)
-    - [L2A](#l2a)
-    - [L2B](#l2b)
-    - [L4A](#l4a)
-- [Running a GEDI Subsetting DPS Job](#running-a-gedi-subsetting-dps-job)
-  - [Submitting a DPS Job](#submitting-a-dps-job)
-  - [Checking the DPS Job Status](#checking-the-dps-job-status)
-  - [Getting the DPS Job Results](#getting-the-dps-job-results)
-- [Getting the GeoJSON URL for a geoBoundary](#getting-the-geojson-url-for-a-geoboundary)
-- [Contributing](#contributing)
-  - [Development Setup](#development-setup)
-  - [Creating an Algorithm Release](#creating-an-algorithm-release)
-  - [Registering an Algorithm Release](#registering-an-algorithm-release)
-- [Citations](#citations)
+## History
+The GEDI Subsetter tool was developed in response to the release of GEDI data and the subsequent interest from the biomass group. The group expressed interest in studying an entire region, which required obtaining all granules that cover that area. Due to GEDI products being an orbit of the International Space Station (ISS), the challenge was to efficiently subset all granules passing through the area of interest. So while the existing methods would work for a small number of granules, the users were seeking a more streamlined approach that would allow them to efficiently gather their data points without the need to process each file separately.
```

Comment:
Nice summary!

* (minor markdown formatting guideline): always put a blank line after a heading (in this case, after the "History" heading)
* Let's link "biomass group", otherwise nobody knows who we're referring to. Perhaps there's also a better or more specific name for the group.
* "studying an entire region" => "studying various regions"
* "which required obtaining all granules that cover that area" => "which requires obtaining all granules that intersect any given region"
* "seeking a more streamlined" => "seeking a highly scalable, automated"